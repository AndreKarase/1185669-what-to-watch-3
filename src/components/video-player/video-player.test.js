import React from 'react';
import renderer from 'react-test-renderer';

import VideoPlayer from './video-player.jsx';

const mock = {
  poster: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  preview: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
};

it(`Should render VideoPlayer`, () => {
  const {preview, poster} = mock;

  const tree = renderer.create(<VideoPlayer
    src={preview}
    poster={poster}
    isPlaying={false}
    muted={false}
  />, {
    createNodeMock: () => {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
