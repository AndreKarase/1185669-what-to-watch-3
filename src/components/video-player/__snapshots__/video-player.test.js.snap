// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render VideoPlayer 1`] = `
<video
  className="player__video"
  muted={false}
  poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
  src="#"
/>
`;
